src/Behat/Behat/Output/Node/EventListener/JUnit/JUnitFeatureElementListener.php

```diff
@@ -154,40 +118,49 @@ private function captureStepEvent(Event $event)
     }
 
     /**
-     * Prints the feature on AFTER event.
+     * Captures scenario tested event.
      *
      * @param Formatter $formatter
-     * @param Event     $event
+     * @param Event $event
      */
-    public function printFeatureOnAfterEvent(Formatter $formatter, Event $event)
+    private function printScenarioEvent(Formatter $formatter, Event $event)
     {
-        if (!$event instanceof AfterFeatureTested) {
+        if (!$event instanceof AfterScenarioTested) {
             return;
         }
+        $afterScenarioTested = $event;
+        $this->scenarioPrinter->printOpenTag(
+            $formatter,
+            $afterScenarioTested->getFeature(),
+            $afterScenarioTested->getScenario(),
+            $afterScenarioTested->getTestResult(),
+            $event->getFeature()->getFile()
+        );
+
+        /** @var AfterStepSetup $afterStepSetup */
```

Comment:
This should be useless thanks to the phpdoc on the property describing the type of items in the array (which is currently described as `AfterSetup`, not `AfterStepSetup`, so one of the 2 places has an issue IMO.